pkg/shell/shell.scss

```diff
@@ -141,17 +141,25 @@ iframe.container-frame {
 }
 
 #credentials-dialog {
-    div.table-scrollable {
-        max-height: 500px;
-        overflow-y: auto;
-        margin-right: 7px;
-        padding-right: 8px;
-        margin-top: 15px;
-        padding-top: 0px;
+    .pf-c-modal-box__body {
+        // Turn off modal scrolling
+        overflow: visible;
+    }
+
+    .table-scrollable {
+        margin-top: var(--pf-global--spacer--md);
+        // Hacky approximation of content outside the list
+        // (we shouldn't have to do this if the layout would be flex or grid)
+        --content-approximation: 18rem;
+        --screen-minus-content: 100vh - var(--content-approximation);
```

Comment:
I'm not a fan of using a magic number, but I couldn't get the crufty code to behave, so here's a hack that works in the extreme smallest size all the way through desktop. As mobile is completely hosed anyway, this unbreaks it to be a little less bad than it already is, in the worst case scenario.

Reply:
I figured out how to make this less hacky. I'll change and force push.